tutorials/i18n/localization_using_gettext.rst

```diff
@@ -1,179 +1,311 @@
-
 Advantages
 ----------
 
 - gettext is a standard format, which can be edited using any text editor
-  or GUI editors such as `Poedit <https://poedit.net/>`_.
+  or GUI editors such as `Poedit <https://poedit.net/>`_. This can be significant
+  as it provides a lot of facilities for translators, such as marking outdated
+  strings, finding strings that haven't been translated etc. 
+- gettext supports plurals and context.
 - gettext is supported by translation platforms such as
   `Transifex <https://www.transifex.com/>`_ and `Weblate <https://weblate.org/>`_,
   which makes it easier for people to collaborate to localization.
-- Compared to CSV, gettext works better with version control systems like Git,
+- Compared to CSV, gettext files work better with version control systems like Git,
   as each locale has its own messages file.
-- Multiline strings are more convenient to edit in gettext files compared
+- Multiline strings are more convenient to edit in gettext PO files compared
   to CSV files.
 
 Disadvantages
 -------------
 
-- gettext is a more complex format than CSV and can be harder to grasp for
+- gettext PO files have a more complex format than CSV and can be harder to grasp for
   people new to software localization.
-- People who maintain localization files will have to install gettext tools
-  on their system. However, as Godot doesn't use compiled message object files
-  (``.mo``), translators can test their work without having to install
-  gettext tools.
-
-Caveats
--------
+- gettext PO files usually use English as the base language. Translators will then use 
+  this base language to translate to other languages. You could still user other 
+  languages as the base language, but this is not common.
 
-- As Godot uses its own PO file parser behind the scenes
-  (which is more limited than the reference GNU gettext implementation),
-  some features such as pluralization aren't supported.
+Getting started with translation using PO files
+-----------------------------------------------
 
-Installing gettext tools
-------------------------
+In this example, we'll create a small example project to show the entire process of translation with PO files.
+The whole process could be summarized as: 
 
-The command line gettext tools are required to perform maintenance operations,
-such as updating message files. Therefore, it's strongly recommended to
-install them.
+::
 
-- **Windows:** Download an installer from
-  `this page <https://mlocati.github.io/articles/gettext-iconv-windows.html>`_.
-  Any architecture and binary type (shared or static) works;
-  if in doubt, choose the 64-bit static installer.
-- **macOS:** Use `Homebrew <https://brew.sh/>`_ to install gettext with the
-  ``brew install gettext`` command.
-- **Linux:** On most distributions, install the ``gettext`` package from
-  your distribution's package manager.
+    Mark strings to translate -> Generate POT -> Translate PO -> Import PO
```

Comment:
It might be better to format this as a list:

```suggestion
- Mark strings to translate.
- Generate a PO template (POT).
- Copy the template into a PO file and translate it.
- Import the PO file into the project.
```